app/views/internal/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="markdown">
+  <% if params[:success] %>
```

Comment:
I would avoid having the views access `params` directly and instead set `@success` and `@pending` instance vars in the controller